tracardi/process_engine/action/v1/strings/string_join/plugin.py

```diff
@@ -0,0 +1,77 @@
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Form, FormGroup, FormField, FormComponent, \
+    Documentation, PortDoc
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.result import Result
+from .model.configuration import Configuration
+
+
+def validate(config: dict) -> Configuration:
+    return Configuration(**config)
+
+
+class JoinAction(ActionRunner):
+    config: Configuration
+
+    async def set_up(self, init):
+        self.config = validate(init)
+
+    async def run(self, payload: dict, in_edge=None) -> Result:
+        dot = self._get_dot_accessor(payload)
+        string = dot[self.config.string]
+        result = self.config.delimiter.join(string)
+        return Result(port="payload", value={"result": result})
+
+
+def register() -> Plugin:
+    return Plugin(
+        start=False,
+        spec=Spec(
+            module='tracardi.process_engine.action.v1.strings.string_join.plugin',
+            className='JoinAction',
+            inputs=["payload"],
+            outputs=["payload"],
+            version='0.7.2',
+            license="MIT",
+            author="Yusuke Kohatsu",
+            manual="string_join_action",
```

Comment:
You have pointed to the documentation file and this is great. If you could place the string_join_action.md in the tracardi-api project in the folder: docs/flow/actions and fill it with the simple documentation that would be great.

Reply:
Will do it. Let me have time to work on please.

Reply:
Super thanks. I will make it a separate issue. So I can merge this one.